### ISSUE TITLE
Couchbase community 4.5.0 erlang/beam.smp eating CPU

### ISSUE THREAD

**Issue by mvysny** [couchbase/goxdcr#4]
I am running a docker version of Couchbase community, as follows:

docker run --rm -ti --net=couchbase --name couchbase-server -p 127.0.0.1:8091-8094:8091-8094 -p 127.0.0.1:11210:11210 couchbase/server:community-4.5.0

The host OS is a 64-bit Ubuntu 16.10. The server has been configured with default values. There are two buckets, default (which is completely empty) and the beer sample database. The erlang process is constantly eating ~30% of one CPU core even though the Couchbase server has been sitting idle with no inbound requests for at least an hour (this is just a testing instance so I'm pretty sure it does nothing).

Attaching a screenshot of htop confirming the issue.
![screenshot from 2016-12-07 10-09-26](https://cloud.githubusercontent.com/assets/254091/20959797/927a7596-bc65-11e6-8950-7ebdbf347de9.png)

**Comment by lastmetroid** [comment]
I see the same behavior:

http://i.imgur.com/pJ3eOK9.png

What is this? Permanent 2 x 3% idle on a xeon 4core is a lot, what is causing this? There are no connections on the couchdb right now.

**Comment by serragnoli** [comment]
Hi @makedir.
Did you get to the bottom of this issue?

**Comment by lastmetroid** [comment]
Nope. Erlang still draining CPU of about 2-3% permanently, even on a brand new Ubuntu Server installation.

**Comment by hubyhuby** [comment]
Exact same behaviour on my Ubuntu 17.04. 64 bits and Couchbase 4.5.1 // CPU i5 HQ
I have the sample databases with a full text search index; sitting IDLE and it takes 20-30% of the CPU all the time. (I  noticed because I am running the server on my laptop and it gets hotter)
I am evaluating aswell, but am wondering if an empty database with no request placed, takes 30% of a moderne CPU....what would it be with 10 requests ;)

I think at the very first start the behaviour was normal, within 2-3%

**Comment by hubyhuby** [comment]
![couchbugcpuidle](https://user-images.githubusercontent.com/26340942/29742599-92d2dbaa-8a82-11e7-95f9-360c9787ca37.jpg)
PS: @makedir , 2-3% is something normal. ElasticSearch is also at 3% IDLE. But 30% IDLE is not something habitual. If I have 3 processes like this on my server, I can shut it down.

**Comment by lastmetroid** [comment]
No... that's not NORMAL @hubyhuby Normal is, when idle takes 0% CPU. I will just remove this damn piece of a crap software I guess. It mostly wont ever get fixed. I see this bug back in reports going back now 5years or longer.

**Comment by xiaoyaozi5566** [comment]
I saw the same issue. beam.smp eating up tons of CPU. Can anyone help?

**Comment by hubyhuby** [comment]
Dear @xiaoyaozi5566,
I came to the conclusion that if you want the real thing without the bug you have to go with the pro version... They don t seem to give much love to the free version unfortunately... They don t do much updates on thise one

**Comment by grreeenn** [comment]
Same here.
Clean CE 5.0 installation wit one EMPTY bucket. No data, no connections to DB, just a clean test installation on my laptop.
Here's the screenshot from the Couchbase UI itself
![cb_cpu_idle](https://user-images.githubusercontent.com/13204857/38782306-13ee5fd8-40fa-11e8-87a9-e69b6f2d0983.png)

Any update?

**Comment by grreeenn** [comment]
OK, found [this](https://issues.couchbase.com/browse/MB-27274) on Couchbase JIRA; see Dave's comment.
